WordPress/gutenberg
# Allow paste of multilines on the title.

Fixes: https://github.com/wordpress-mobile/gutenberg-mobile/issues/1124

## Description


Updated the paste code for the title component in RN to be able to support multiline pastes.
The first line is pasted to the title and the follow up lines are added as blocks in the content.
This matches the current behavior on the web.

## How has this been tested?




This can be tested using the native demo app by running `npm run native ios` and do the following steps:

 - Start the demo app
 - Copy from another app multiple lines of text
 - Paste on the title
 - Check if the first line is added to the title and the following ones added as blocks in the content.

## Screenshots 

## Types of changes





Matches functionality between the web and native versions.

## Checklist:
